app/models/druid_version_zip.rb

```diff
@@ -10,11 +12,27 @@ def initialize(druid, version)
     @version = version
   end
 
+  # Changes directory so that the storage root (and druid tree) are not part of
+  # the archival directory structure, just the object.
```

Comment:
What do you think about adding an example of the directory structure we're going for here?  The comment assumes the reader has the Moab druid-tree layout readily loaded into their human memory.

Also, what do you think about starting the comment "Creates zip for Moab version, but changes ..."  The comment as it stands reads a little orthogonal to the name of the method.

Reply:
IMO, I'm not assuming the reader knows anything about druid trees.  I'm saying only the object is preserved and no other structure.  

I agree this comment is more about the `chdir` than a description of the method itself, so I'll add a broader description.